torch/onnx/symbolic_opset9.py

```diff
@@ -1612,3 +1612,109 @@ def gather(g, self, dim, index, sparse_grad=False):
     index = g.op("Cast", g.op("OneHot", index, depth, values, axis_i=dim), to_i=sym_help.cast_pytorch_to_onnx[dtype])
     mul = g.op("Mul", g.op("Unsqueeze", self, axes_i=[dim + 1]), index)
     return g.op("ReduceSum", mul, axes_i=[dim], keepdims_i=0)
+
+
+def index(g, self, index):
+    if sym_help._is_packed_list(index):
```

Comment:
The symbolic of index op incurs some internal pipe breakage.
The breakage is caused by the Nonzero operator. We don't have it in Caffe2.

At least we have two options so far:
1) implement the Nonzero op in Caffe2.
2) or add check here, if the export mode is ONNX_ATEN_FALLBACK, we direct emit aten operator

Reply:
Thanks for the suggestions! Updated the PR to extend Caffe2 backend NonZero with ATen::nonzero.